Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessCluster.java

```diff
@@ -517,6 +533,63 @@ public AzureHttpObservable getHttp() {
                 .defaultIfEmpty(this);
     }
 
+    @Nullable
+    private UpdateSparkResourcePool preparePatchResourcePool(int workerTargetInstanceCount) {
+        if (master == null || worker == null) {
```

Comment:
I did find the reference to `master` or `worker`, better to move them out and annotate the function as `@Notnull`